core/indexer/elasticsearchDatabase.go

```diff
@@ -213,32 +243,60 @@ func (esd *elasticSearchDatabase) buildTransactionBulks(
 	return bulks
 }
 
-func (esd *elasticSearchDatabase) serializeBulkTx(bulk []*Transaction) bytes.Buffer {
-	var buff bytes.Buffer
-	for _, tx := range bulk {
-		meta := []byte(fmt.Sprintf(`{ "index" : { "_id" : "%s", "_type" : "%s" } }%s`, tx.Hash, "_doc", "\n"))
-		serializedTx, err := json.Marshal(tx)
+// SaveMiniblocks will prepare and save information about miniblocks in elasticsearch server
+func (esd *elasticSearchDatabase) SaveMiniblocks(header data.HeaderHandler, body *block.Body) {
+	miniblocks := esd.getMiniblocks(header, body)
+	if miniblocks == nil {
+		log.Warn("indexer: could not index miniblocks")
+		return
+	}
+	if len(miniblocks) == 0 {
+		return
+	}
+
+	buff := serializeBulkMiniBlocks(header.GetShardID(), miniblocks)
+	err := esd.dbWriter.DoBulkRequest(&buff, miniblocksIndex)
+	if err != nil {
+		log.Warn("indexer", "error1", "indexing bulk of miniblocks",
```

Comment:
why not `log.Warn("indexing bulk of miniblock", "error", err)` ?

Reply:
done.  refactored